superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -402,21 +404,21 @@ function DashboardList(props: DashboardListProps) {
     },
     {
       Header: t('Status'),
-      id: 'published',
+      id: 'id',
       input: 'select',
-      operator: FilterOperators.equals,
+      operator: FilterOperator.status,
       unfilteredLabel: t('Any'),
       selects: [
-        { label: t('Published'), value: true },
-        { label: t('Unpublished'), value: false },
```

Comment:
Bycatch, change the filter select to be consistent with elsewhere:

<img width="417" alt="filter-select" src="https://user-images.githubusercontent.com/335541/115085359-43e5ae00-9ebf-11eb-90b6-b6b7d5fb598a.png">

Reply:
makes sense!